applications/inventory-app/public/inventory.js

```diff
@@ -0,0 +1,293 @@
+}
+
+// END SEND AND RECEIVE SETTINGS STATE
+
+function saveInventory() {
+    Settings.setValue(inventoryDataSettingString, inventoryData);
+}
+
+function loadInventory() {
+    inventoryData = Settings.getValue(inventoryDataSettingString);
+}
+
+function saveSettings() {
+    Settings.setValue(inventorySettingsString, inventorySettings);
+}
+
+function loadSettings() {
+    inventorySettings = Settings.getValue(inventorySettingsString);
+}
+
+function pushReceivedItemToQueue(senderUUID, type, name, url) {
+    console.info("Receiving an item:", name, "from:", senderUUID);
+    var getAvatarData = AvatarList.getAvatar(senderUUID);
+    var senderName = getAvatarData.sessionDisplayName;
+    var senderDistance = Vec3.distance(MyAvatar.position, getAvatarData.position);
+    
+    var packageRequest = {
+        "senderUUID": senderUUID,
+        "senderName": senderName,
+        "senderDistance": senderDistance,
+        "data": {
+            "type": type,
+            "name": name,
+            "url": url
+        }
+    }
+    
+    if (receivingItemQueue.length === RECEIVING_ITEM_QUEUE_LIMIT) {
+        receivingItemQueue = receivingItemQueue.slice(1, 5);
+    }
+    
+    receivingItemQueue.push(packageRequest);
+    ui.messagesWaiting(receivingItemQueue.length > 0);
+}
+
+function sendReceivingItemQueue() {
+    sendToWeb("script-to-web-receiving-item-queue", receivingItemQueue);
+}
+
+function updateReceivingItemQueue(data) {
+    receivingItemQueue = data;
+    ui.messagesWaiting(receivingItemQueue.length > 0);
+}
+
+function sendNearbyUsers() {
+    var nearbyUsers = AvatarList.getAvatarsInRange(MyAvatar.position, 25); // Find all users within 25m.
+    var nearbyUsersToSend = [];
+    
+    nearbyUsers.forEach(function(user, i) {
+        var objectToWrite;
+        var aviDetails = AvatarList.getAvatar(user)
+        var aviName = aviDetails.displayName;
+        var aviDistance = Vec3.distance(MyAvatar.position, aviDetails.position);
+        // Window.alert("aviName" + aviName + "user" + user + "MyAvatar.sessionUUID" + MyAvatar.sessionUUID);
+        if (user != MyAvatar.sessionUUID || Controller.getValue(Controller.Hardware.Keyboard.Shift)) { // Don't add ourselves to the list!
+            objectToWrite = { "name": aviName, "distance": aviDistance, "uuid": user };
+            nearbyUsersToSend.push(objectToWrite);
+        }        
+    });
+
+    sendToWeb("script-to-web-nearby-users", nearbyUsersToSend);
+}
+
+function useItem(item) {
+    
+    //TODO: Add animation support for avatars...?
+    
+    // Convert the item.type before checking it...
+    item.type = item.type.toUpperCase();
+    
+    // Depending on the type, we decide how to load this item.
+    if (item.type == "SCRIPT") {
+        ScriptDiscoveryService.loadScript(item.url, true, false, false, true, false); // See SDS.loadScript in APIDocs for more.
+    }
+    
+    if (item.type == "MODEL") {
+        var entityID = Entities.addEntity({
```

Comment:
```suggestion
        Entities.addEntity({
```
`entityID` is assigned a value but never used.